CONTRIBUTING.md

```diff
@@ -0,0 +1,338 @@
+# Apache APISIX Blog Contributing Guide
+
+## Overview
+
+Welcome to Apache APISIX's website and blog repository.
+
+This guideline will teach you how to contribute if you want to add a new blog, or modify existing blog contents.
+
+If you find an issue on current documents, please feel free to file an issue to let the community know about it, or you can sumbit a pull request to fix or update. Both actions are welcome and recommended.
+
+The blogs are written in two languages: English and Chinese. We encourage contributors to add blogs in their preferred language. It is completely up to you. We can handle translations and ask you to do a pull request review later.
+
+English blogs are located in the `website/blog` directory, in which they are categorized by year, month and date. For example, `website/blog/2021/11/22/develop-apisix-ingress-with-nocalhost-in-kubernetes.md` means that a blog named `develop-apisix-ingress-with-nocalhost-in-kubernetes.md` was published on November 22nd, 2021, and it is located in the `website/blog/2021/11/22 directory`.
+
+Similarly, Chinese blogs are located in website/i18n/zh/docusaurus-plugin-content-blog directory and follow the same rules described above.
+
+## What you can contribute
+
+### Add a new blog
+
+To add a new blog, please perform the following steps:
+
+1. Find the right place to store your blog.
+    1. If you plan to submit a blog written in English, please create a markdown file under `website/blog` directory.
+    2. If you plan to submit a blog written in Chinese, please create a markdown file under `website/i18n/zh/docusaurus-plugin-content-blog` directory.
+    3. If you cannot find an existing year, month, or date directory that fits your desired published date, you can go ahead and create such directory on your own.
+
+2. Create a markdown file in the directory. Please make sure that the file name is written in English with no capitalized letters. During review session, we may suggest changing it to achieve better SEO performance. (Yes, currently some files names contain capitalized letters, we are working on it, see https://github.com/apache/apisix-website/issues/713)
+
+3. Add text, images, diagrams, charts to the markdown file.
+    1. There is nothing much to say about adding text.
+    2. To add images, please upload images to https://markdown.apiseven.com/#/ , and copy the links from there.
+    3. To add charts or diagrams, we are happy to see any sort of charts and diagrams, as long as it formats correctly on the page. From previous experience, charts with 4 columns or less are the perfect size displayed on screen.
+
+4. Optional: run locally to inspect any typos or formatting issues left behind. We highly recommend running the website locally before submission. Although we implemented lint and error checks in the repository, we suggest running it locally to avoid repeated work. To run it locally, please run `cd website`, and then run `yarn start` commands in your terminal.
```

Comment:
```suggestion
4. Optional: run locally to inspect any typos or formatting issues left behind. Although we implemented lint and error checks in the repository, we suggest running it locally to avoid repeated work. To run it locally, please run `cd website`, and then run `yarn start` commands in your terminal.
```